### ISSUE TITLE
Header property validations-

### ISSUE THREAD

**Issue by SameerSiddiqui** [apigee-127/swagger-tools#440]
I just noticed there is no way to define acceptable Mime Types in header-

It either applies to all or none - 

produces:
  - application/json
  - application/pdf

That goes to root level.

How can i make sure that for a given end point i enforce my client to send me a proper value for header property such as accept: application/pdf  and if they send me any thing else, swagger tool catches it.

As of now i have to write custom logic, explicity check  the accept value inside header and throw error back. 

Thanks,
Sameer

**Comment by whitlockjc** [comment]
This should already be working.  Can you give more detail on how to reproduce this?

**Comment by fastcodejava** [comment]
Seems like to be a valid issue! I tried this :


```
  /products/abcd:
    x-swagger-router-controller: aaa
    post:
      summary:       summary
      operationId: xyz
      **produces**:
        - **application/pdf**
       parameters:

```

I tried a post with Accept as `application/json`, it seems to go right through.

**Comment by SameerSiddiqui** [comment]
Yes- @fastcodejava  i did exactly the same- Thanks for posting an example. 

So basically, i am trying to put a validation on accept:application/pdf on one of the rest call-
